test/e2e/deploy/volumeleaks.go

```diff
@@ -48,8 +49,21 @@ func GetHostVolumes(d *Deployment) map[string][]string {
 		// Intentional Output instead of CombinedOutput to dismiss warnings from stderr.
 		// lvs may emit lvmetad-related WARNING msg which can't be silenced using -q option.
 		out, err := ssh.Output()
-		if err != nil && os.IsNotExist(err) {
-			break
+		if err != nil {
+			if os.IsNotExist(err) {
+				// ssh wrapper does not exist: all nodes handled.
+				break
+			}
+			ginkgo.By(fmt.Sprintf("ssh to node %d failed: %v, retry", worker, err.Error()))
+			for retry := 1; retry < 4; retry++ {
```

Comment:
Wouldn't it be better to merge the first attempt into the loop body?

When ssh gets repeated, why does it not need to check for `os.IsNotExist`?

Reply:
> When ssh gets repeated, why does it not need to check for `os.IsNotExist`?

IsNotExist is there to detect end of nodes. I assume that set of ssh handlers does not change during running this. We check for IsNotExist above repeat loop, and that means (if assumption above is true) that also in repeated attempts, the ssh handler should be there, therefore we dont check for existence error.

Reply:
> Wouldn't it be better to merge the first attempt into the loop body?

makes sense, will avoid repeated code lines. And that would also make the IfNotExist checking more uniform. I will change.